VoodooI2CHID/VoodooI2CMultitouchHIDEventDriver.cpp

```diff
@@ -468,6 +474,28 @@ void VoodooI2CMultitouchHIDEventDriver::handleStop(IOService* provider) {
     super::handleStop(provider);
 }
 
+UInt32 VoodooI2CMultitouchHIDEventDriver::parseElementPhysicalMax(IOHIDElement* element) {
+    UInt32 physical_max = element->getPhysicalMax();
+
+    UInt8 raw_unit_exponent = element->getUnitExponent();
+    if (raw_unit_exponent >> 3) {
+        raw_unit_exponent = raw_unit_exponent | 0xf0; // Raise the 4-bit int to an 8-bit int
+    }
+    SInt8 unit_exponent = *reinterpret_cast<SInt8*>(&raw_unit_exponent);
+
+    // Scale to 0.01 mm units
+    UInt32 unit = element->getUnit();
+    if (unit == 0x11) {        // Centimeters
```

Comment:
Can we put 0x11 and 0x13 as header definitions as these are constants?

Reply:
Done.